[OC-1473] Upgrade MongoDB and Keycloak docker images.

geppyz opened the pull request:
This branch upgrades the MongoDB docker image from 4.1.1-xenial to 4.4.4-bionic and upgrades the Keycloak docker image from 6.0.1 to 12.0.2 and adds a configuration parameter to support the 'upload_scripts' feature in Keycloak 12.

JeroenGommans reviewed (approved):
Look good. Note that OpFab already updated to mongo:4.4.4-bionic

freddidierRTE commented:
works well on my computer 
Could you  :
- add as well the modification on config/dev/docker-compose.yml
- rebase and squash your commits in one

geppyz commented:
> works well on my computer
> Could you :
> 
> * add as well the modification on config/dev/docker-compose.yml
> * rebase and squash your commits in one

I've squashed the changes to one commit and also updated the dev docker-compose file.
Thanks for the feedback!
